### PR TITLE
Vega E2E automated tests stabilization

### PR DESCRIPTION
**Pull request by re-jas**
https://issues.folio.org/browse/FAT-4842
[Patron block tests](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1252/allure/)

### CODE REVIEW

File: cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js [hunk 1]
```diff
@@ -65,6 +66,10 @@ describe('TLR: Item renew', () => {
   before('Preconditions', () => {
     cy.getAdminToken()
       .then(() => {
+        cy.loginAsAdmin({
```
Comment (IhorBohdan) [inline cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js:69]: You can do it in one line

File: cypress/support/fragments/settings/circulation/titleLevelRequests.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { Pane, Button, Checkbox } from '../../../../../interactors';
+import { Pane, Button, Checkbox, Callout, calloutTypes } from '../../../../../interactors';
+import InteractorsTools from '../../../utils/interactorsTools';
```
Comment (IhorBohdan) [inline cypress/support/fragments/settings/circulation/titleLevelRequests.js:2]: Fix linter issues

File: cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js]: done

File: cypress/support/fragments/settings/circulation/titleLevelRequests.js (hunk 2 above)
Comment (re-jas) [reply cypress/support/fragments/settings/circulation/titleLevelRequests.js:2]: done

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js [hunk 3]
```diff
@@ -204,6 +202,7 @@ describe('TLR: Item renew', () => {
     cy.wrap(instanceData.itemIds).each((item) => {
       cy.deleteItemViaApi(item);
     });
```
Comment (penfriend) [inline cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js:204]: Let's have these two tests related to TLR in one file with one describe.

Comment (re-jas) [reply cypress/e2e/loans/item-assigned-recall-not-renewable.cy.js]: done

**Review by penfriend — APPROVED** [review]

**Pull request merged**